ui/lib/pki/addon/components/page/pki-issuer-list.hbs

```diff
@@ -3,68 +3,114 @@
+                {{/if}}
+                {{#if (not-eq pkiIssuer.isRoot undefined)}}
+                  <span class="tag has-text-grey-dark" data-test-is-root-tag={{idx}}>{{if
+                      pkiIssuer.isRoot
+                      "root"
+                      "intermediate"
+                    }}</span>
+                {{/if}}
+                {{#if pkiIssuer.serialNumber}}
+                  <span class="tag is-transparent has-right-margin-none" data-test-serial-number={{idx}}>
+                    <InfoTooltip>
+                      Serial number
+                    </InfoTooltip>
+                    {{pkiIssuer.serialNumber}}
+                  </span>
+                {{/if}}
+                {{#if pkiIssuer.parsedCertificate.common_name}}
+                  <span class="tag is-transparent has-left-margin-none" data-test-common-name={{idx}}>
+                    <InfoTooltip>
+                      Common name
+                    </InfoTooltip>
+                    {{pkiIssuer.parsedCertificate.common_name}}
+                  </span>
+                {{/if}}
+              </div>
+            </div>
+          </div>
+          <div class="level-right is-flex is-paddingless is-marginless">
+            <div class="level-item">
+              <PopupMenu>
+                <nav class="menu" aria-label="issuer config options">
+                  <ul class="menu-list">
+                    <li data-test-popup-menu-details>
+                      <LinkTo @route="issuers.issuer.details" @model={{pkiIssuer.id}}>
+                        Details
+                      </LinkTo>
+                    </li>
+                    <li>
+                      <LinkTo @route="issuers.issuer.edit" @model={{pkiIssuer.id}}>
+                        Edit
+                      </LinkTo>
+                    </li>
+                  </ul>
+                </nav>
+              </PopupMenu>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="level-right is-flex is-paddingless is-marginless">
-        <div class="level-item">
-          <PopupMenu>
-            <nav class="menu" aria-label="issuer config options">
-              <ul class="menu-list">
-                <li data-test-popup-menu-details>
-                  <LinkTo @route="issuers.issuer.details" @model={{pkiIssuer.id}}>
-                    Details
-                  </LinkTo>
-                </li>
-                <li>
-                  <LinkTo @route="issuers.issuer.edit" @model={{pkiIssuer.id}}>
-                    Edit
-                  </LinkTo>
-                </li>
-              </ul>
-            </nav>
-          </PopupMenu>
-        </div>
-      </div>
-    </div>
-  </LinkedBlock>
-{{/each}}
\ No newline at end of file
+      </LinkedBlock>
+    {{/each}}
+  </:list>
+  <:empty>
+    <EmptyState @title="PKI not configured" @message={{this.notConfiguredMessage}}>
```

Comment:
Moved the empty state into here from the route template, so I needed to add a backing class for `notConfiguredMessage`